Automattic/sensei
# Rename the column title from Description to Email

Resolves #6597 

## Proposed Changes
* Rename column title from Description to Email
* Move Email column to be the first column in the table.

## Testing Instructions

1. Enable 'email_customization' feature flag.
2. Go to Sensei LMS -> Tools -> Re-create Emails.
3. Go to Sensei LMS -> Settings -> Emails and check the appearance of the table.

## Screenshots
<img width="1517" alt="CleanShot 2023-03-07 at 19 05 19@2x" src="https://user-images.githubusercontent.com/329356/223417293-2ad2cfa6-96e8-4d23-8cba-e366db933164.png">
<img width="376" alt="CleanShot 2023-03-07 at 19 05 36@2x" src="https://user-images.githubusercontent.com/329356/223417327-d6735fe7-9e72-4de4-8aab-56e9c653b879.png">


## Pre-Merge Checklist
